src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.CoreCLR.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This assembly is what we call a "partial facade", which I will be creating some documentation for soon. The "CoreCLR" suffix just means that this version of the project is built for CoreCLR. Internally we have a similar project for .NET Native.